### ISSUE TITLE
[Unblock request] please remove my account from bot account list

### ISSUE THREAD

**Issue by huphia** [CommanderRoot/twitch-tools#76]
Hello, my twitch username is 'huphia'. Recently twitch banned my account because some bot hacked my account and spam messages were sent. But after asking twitch, they unbanned me after a day. I think my username is added into your bot account list because I cannot watch some streamers nor follow them, even though some are new to me. Please check the list and remove my username from it. I am not a bot, my account was just hacked. Hope you can remove my username, thankyou.

**Comment by CommanderRoot** [comment]
Duplicate of #30 

Your account was added to the list on 2021-04-28 as it was part of one or multiple follow bot raids. Either way it happened I have now removed it from the list.

**Comment by huphia** [comment]
How long does it take to get unblocked by other streamers? I still see some streams blocked for me.

**Comment by CommanderRoot** [comment]
Removing your account will only prevent further blocks. All streamers which have blocked your account will have to manually unblock you

**Comment by huphia** [comment]
Sigh, so there wasn't much point to the request then. What an irresponsible tool, yikes.

**Comment by CommanderRoot** [comment]
Of course there is a point to get your account removed. As I said it prevents it from be blocked by even more users

**Comment by huphia** [comment]
Yes, that's why I said 'not much point'. Damage has already been done by your tool, which doesn't even let the user know, and you are doing nothing to rectify it properly. I guess minority always get rekt.
